bundles/com.zeligsoft.domain.dds4ccm/src/com/zeligsoft/domain/dds4ccm/utils/ConnectorTypeUtility.java

```diff
@@ -0,0 +1,195 @@
+/**
+ * Copyright 2018 ADLINK Technology Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.zeligsoft.domain.dds4ccm.utils;
+
+import java.io.IOException;
+import java.util.List;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.uml2.uml.Element;
+import org.eclipse.uml2.uml.Model;
+import org.eclipse.uml2.uml.Package;
+import org.eclipse.uml2.uml.PackageImport;
+import com.ibm.xtools.modeler.ui.UMLModeler;
+import com.zeligsoft.base.zdl.staticapi.util.ZDLFactoryRegistry;
+import com.zeligsoft.base.zdl.util.ZDLUtil;
+import com.zeligsoft.domain.dds4ccm.Activator;
+import com.zeligsoft.domain.dds4ccm.ConnectorType;
+import com.zeligsoft.domain.idl3plus.IDL3PlusNames;
+import com.zeligsoft.domain.idl3plus.api.Connectors.ConnectorDef;
+import com.zeligsoft.domain.idl3plus.api.Generics.TemplateModule;
+import com.zeligsoft.domain.omg.corba.CORBADomainNames;
+import com.zeligsoft.domain.omg.corba.api.IDL.CORBAModule;
+import com.zeligsoft.domain.omg.corba.api.IDL.CORBAModuleContained;
+
+public class ConnectorTypeUtility {	
+	
+	/**
+	 * Retrieves pathmap of a connector library
+	 * 
+	 * @param preferredConnectorType
+	 *          Preferred value of 'ConnectorType' to be used during migration 
+	 *            
+	 * @return pathmap of a connector library which corresponds to the input connectorType value  
+	 *         
+	 */
+	
+	static String getPathmapConnectorLib(String preferredConnectorType){
```

Comment:
All the methods in this class are package private (i.e. they have no 'public', 'private' or 'protected' qualifier). This means that no method outside the current package can all them. Are any of these methods candidates for a 'public' qualifier?